sunpy/timeseries/sources/eve.py

```diff
@@ -75,21 +75,26 @@ def plot(self, axes=None, **kwargs):
         """
         self._validate_data_for_plotting()
         names = ("Flux \n 0.1-7nm", "Flux \n 18nm", "Flux \n 26nm", "Flux \n 30nm", "Flux \n 36nm")
-        colors = ('tab:blue', 'tab:orange', 'tab:green', 'tab:red', 'tab:purple', 'tab:brown', 'tab:pink', 'tab:gray', 'tab:olive', 'tab:cyan')
+        colors = ('tab:blue', 'tab:orange', 'tab:green', 'tab:red', 'tab:purple',
+                  'tab:brown', 'tab:pink', 'tab:gray', 'tab:olive', 'tab:cyan')
         predefined_axes = False
         if isinstance(axes, np.ndarray):
             predefined_axes = True
         elif axes is None:
             axes = self.to_dataframe().plot(subplots=True, sharex=True, **kwargs)
         for i, ax in enumerate(axes):
             if predefined_axes:
-                ax.plot(self._data[self._data.columns[i]], color=colors[i%len(colors)], label=self._data.columns[i])
+                ax.plot(self._data[self._data.columns[i]], color=colors[i %
+                        len(colors)], label=self._data.columns[i])
```

Comment:
We should change the line length in setup.cfg

Reply:
This is a really weird line break. Is that from pre-commit?

Reply:
This is autopep8.